todo_app/app.py

```diff
@@ -1,20 +1,39 @@
 from flask import Flask, render_template, request, redirect, url_for
-from todo_app.data.session_items import get_items, add_item
-from todo_app.flask_config import Config
+# from todo_app.flask_config import Config
+import todo_app.trello_client as trello_client
+from todo_app.view_model import ViewModel
+from todo_app.data.todo_item import TodoItem
 
 app = Flask(__name__)
-app.config.from_object(Config)
+# app.config.from_object(Config)
```

Comment:
If we're not using this code, let's get rid of it

Reply:
done